nf_core/pipelines/create/create.py

```diff
@@ -101,6 +101,7 @@ def __init__(
                 ".prettierrc.yml",
                 ".github/workflows/fix-linting.yml",
             ],
+            "citations": ["CITATIONS.md", "assets/methods_description_template.yml"],
```

Comment:
I don't think we should remove citations.md. I think it is similar to a license file and helps keeping track of nf-core in the wider world (both for us, but also when people publish their workflows)

Reply:
We have the mention to cite the nf-core publication in the readme, would this be enough?
(note for myself: there are some mentions to `CITATIONS.md` in the readme!)

Otherwise, would it be better to exclude the citations to the modules tools from `CITATIONS.md` and keep only nf-core and nextflow?

Reply:
yes, the modules tools should also be dependent on their selection, but that is for a different PR.